src/raftstore/store/worker/region.rs

```diff
@@ -162,14 +163,14 @@ impl PendingDeleteRanges {
         ranges
     }
 
+    fn remove(&mut self, start_key: &[u8]) -> Option<(u64, Vec<u8>, Vec<u8>)> {
```

Comment:
Is the return value needed?

Reply:
I think so, it's used for `assert` something.